iree/hal/drivers/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@
 
 # Doesn't use bazel_to_cmake because of custom configuration vars
 
+if (NOT ${IREE_ENABLE_THREADING})
```

Comment:
This could at least use a comment (why does using the set of drivers require threading?).

Reply:
It's the implicit dependency of threading through `abseil` 
https://github.com/abseil/abseil-cpp/blob/6f9d96a1f41439ac172ee2ef7ccd8edf0e5d068c/CMakeLists.txt#L93

I will add comments and todos (related to github.com/google/iree/issues/3848)